tests/install/test.py

```diff
@@ -78,7 +125,17 @@ def test_install(self, firmware, create_vms, iso_remaster, iso_version, local_sr
         mac_address = vif.param_get('MAC')
         logging.info("Host VM has MAC %s", mac_address)
 
-        host_vm.insert_cd(iso_remaster)
+        if source_type == "pxe":
+            # FIXME: don't hard code the iso_key...
+
+            #marker = request.node.get_closest_marker("installer_iso")
+            #param_mapping = marker.kwargs.get("param_mapping", {})
+            #(iso_key, _) = callable_marker(marker.args[0], request, param_mapping=param_mapping)
+            iso_key="xcpng-8.3-rc1"
+
+            setup_pxe_boot(host_vm, mac_address, iso_key)
```

Comment:
Here maybe I should add a method in vm.py and do something like
`host_vm.pxe_boot(setup_pxe_boot)` and declare setup_pxe_boot in _tests/install/conftests.py_ ?
So I will be able to get the iso_key using the custom marker... @ydirson what do you think?

Reply:
But I'm not sure because setup_pxe_boot is using data.py info to setup PXE so I don't see it as a method in `vm.py`...

Reply:
to be considered after the deployment to PXE unpacks a remastered ISO